packages/cli/lib/commands/build.js

```diff
@@ -32,4 +34,83 @@ module.exports = async function (src, argv) {
 	if (argv.json) {
 		await runWebpack.writeJsonStats(stats);
 	}
+}
+
+const options = [
+	{
+		name: '--src',
+		description: 'Specify source directory',
+		default: 'src',
+	},
+	{
+		name: '--dest',
+		description: 'Specify output directory',
+		default: 'build',
+	},
+	{
+		name: '--cwd',
+		description: 'A directory to use instead of $PWD',
+		default: '.',
+	},
+	{
+		name: '--esm',
+		description: 'Builds ES-2015 bundles for your code',
+		default: true,
+	},
+	{
+		name: '--sw',
+		description: 'Generate and attach a Service Worker',
+		default: true,
+	},
+	{
+		name: '--babelConfig',
+		description: 'Path to custom Babel config',
+		default: '.babelrc',
+	},
+	{
+		name: '--json',
+		description: 'Generate build stats for bundle analysis',
+	},
+	{
+		name: '--template',
+		description: 'Path to custom HTML template',
+	},
+	{
+		name: '--preload',
+		description: 'Adds preload tags to the document its assets',
+		default: false,
+	},
+	{
+		name: '--analyze',
+		description: 'Launch interactive Analyzer to inspect production bundle(s)',
+	},
+	{
+		name: '--prerenderUrls',
```

Comment:
We allow `--prerender-urls` too right?

Reply:
I don't know if camel case is really something a cli should use, but seems like that's what we had before and only that

Reply:
seems like we don't allow `--prerender-urls` as of now. So technically changing this would be a breaking change.